.github/workflows/generate-pinval.yaml

```diff
@@ -34,7 +35,7 @@ on:
           - Development bucket
           - Production bucket
         default: Development bucket
-        required: false
+        required: true
```

Comment:
I don't think this change makes a material difference, since the GitHub UX seems to make it impossible to null out a `choice` variable, but I figured we might as well mark this variable as required to hint at the fact that it's necessary.